src/app/viewer/useStorage.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
```

Comment:
this file should probably be called useCountryStorage.

Reply:
the function could be called `useCountryStorage` aswell ; useStorage is not precise enough

Reply:
@raphoester 
Name changed [here](src/app/viewer/useCountryStorage.ts).